### DISCUSSION TITLE
MQTT SmartHome

### DISCUSSION THREAD

**Discussion by pherting** [Wendelpunkt/ToypadChallenge#22] [Ideas]
Hello,

i love my SmartHome, i love my HomeAssistant and i love MQTT. Why not combine it togther an create a Plugin to Trigger some topics. Than you can easy Trigger some acrions in HomeAssistant.

So when someone loves the Idea, i want to create this Plugin für the Community! 
I dont Code in c# since a while, more nodeJS the Last years, but for you i start visual Studio again 😂

Let me know your ideas and Feedback about it.

Thanks ❤️

**Comment by tomwendel** [comment]
I would love it! So the toypad would allow me to setup different light scenes bound to specific LEGO figures. I could use this in my office to configure a day theme, night theme and stream theme.

Best place to put another projekt in a different language would be the src-folder. just create a subfolder named after your project

**Comment by tschissler** [comment]
@pherting, I love the idea of triggering MQTT topics with the toypad.
If you need a hand with this, let me know. I have some experience with C# and MQTT and would love to do some collaboration or even pair programming :-)

**Reply by pherting** [reply]
That Sounds great!!! Of course we can do it togther. I started before 2 days with it, but i can Show you my stuff and we can make it finally perfect 😃 Do you have Discord? Iam on toms Discord, my Name ist bi0s. You can write me there private.
